scripts/martech.js

```diff
@@ -123,10 +124,42 @@ function setMarTechContext() {
  */
 
 function setDigitalData() {
+  // creates a string from the categories & products for analytics
+  // example: "Category: #AdobeForAll | Category: Adobe Life | Product: Photoshop"
+  const getPageFilterInfo = () => {
+    let pageFilterInfo = "";
+    const categories = window.blog.topics || [];
+    const products = window.blog.products || [];
```

Comment:
By using `window.blog.topics`/ `window.blog.products`, you will only track `Category: #AdobeForAll | Category: Adobe Life | Product: Photoshop`
If you use `window.blog.allTopics` / `window.blog.allProducts`, you would track: `Category: #AdobeForAll | Category: #Content Series | Category: Adobe Life | Product: Creative Cloud | Product: Photoshop`

Explanation:
- `window.blog.topics` / `window.blog.products` contain the "leaf" topics / products defined in the taxonomy and which are the values stored on each articles.
- `window.blog.allTopics` / `window.blog.allProducts` contains the "leaf" and their resolved "parent" topics as defined in the taxonomy.

I am not sure if this is the intent and this may have an impact on the reporting: if you store only the "leaf" topics / products, you would not easily be able to report on all products from the Creative Cloud for example (reporting tool should add `product = photoshop, premiere...` to the query, while if you use the lead and parent topics, you can easily query on `product = creative cloud` or `product = photoshop`).

Subtile but may have an impact.

Reply:
Thanks for the in depth explanation. I was wondering about that but wasn't sure about the background between both.

Switched to `blog.allTopics/blog.allProducts`